thirdparty/kpvcrlib/CMakeLists.txt

```diff
@@ -71,6 +72,7 @@ add_definitions(
     -DALLOW_KERNING=1
     -DCR3_PATCH=1
     -DMATHML_SUPPORT=1
+    -DUSE_STD_REGEX=1
```

Comment:
This should now be called `-DUSE_SRELL` (and moved up with the other `-DUSE_*`).
Also update the `#if USE_STD_REGEX` in crengine.